src/i18n/en-actions.ts

```diff
@@ -78,8 +80,8 @@ const actionsMessageDescriptors = {
       ${ColonyActionType.EmitDomainReputationReward} {Award}
       ${ColonyActionType.EmitDomainReputationRewardMotion} {Award}
       ${ColonyActionType.CreateDecisionMotion} {Decision}
-      ${ColonyActionType.AddVerifiedMembers} {Manage verified members}
-      ${ColonyActionType.RemoveVerifiedMembers} {Manage verified members}
+      ${ColonyActionType.AddVerifiedMembersMotion} {Manage verified members}
+      ${ColonyActionType.RemoveVerifiedMembersMotion} {Manage verified members}
```

Comment:
Think this was meant to be an addition rather than a change!

Reply:
how did this even happen, mea culpa :facepalm: but nice catch!